source/includes/_errors.md

```diff
@@ -14,6 +24,7 @@ Error Code | Meaning
 410 | Gone -- The kitten requested has been removed from our servers
 413 | Request Entity Too Large -- You sent too much data to us
 418 | I'm a teapot
+422 | Unprocessable Entity
```

Comment:
I've done some research on this status and it seems to be similar to `400` (what we use now for validation errors). I prefer the latter for some reasons:
 - it's consistent with other api calls
 - 422 comes from the WebDav extension, 400 was born in the core http, so it seems easier to support this one

Do you have any strong preferences against 400?